static/js/components/LearningResourceCard.js

```diff
@@ -68,62 +106,47 @@ export const LearningResourceCard = ({
     })
 
   return (
-    <div className="learning-resource-card">
-      <div className="card-contents">
-        <div className="cover-image" onClick={showResourceDrawer}>
-          <img
-            src={embedlyThumbnail(
-              SETTINGS.embedlyKey,
-              object.image_src || "",
-              CAROUSEL_IMG_HEIGHT,
-              CAROUSEL_IMG_WIDTH
-            )}
-            height={CAROUSEL_IMG_HEIGHT}
-            alt={`cover image for ${object.title}`}
-          />
+    <Card
+      className={getClassName(searchResultUI)}
+      borderless={searchResultUI === SEARCH_GRID_UI}
+    >
+      {searchResultUI === SEARCH_GRID_UI ? (
+        <CoverImage object={object} showResourceDrawer={showResourceDrawer} />
+      ) : null}
+      <div className="lr-info">
+        <div className="row resource-type">
+          {readableLearningResources[object.object_type]}
```

Comment:
Comparing the mockups vs this, it looks like the resource type text has a higher font-weight (`600`) than other text

Reply:
I set that initially but when I did it looked very visually different than the mockups, not sure why. here's what's in the mockups:

![fromthemockups](https://user-images.githubusercontent.com/6207644/63540705-9607dd80-c4ea-11e9-9d66-b88edeff9480.png)

this is how it looks with the styling from the 'inspect' mode on invision applied (14pt font, 600 weight text):

![withmockupstyles](https://user-images.githubusercontent.com/6207644/63541164-81781500-c4eb-11e9-8ad9-997b3427f9eb.png)


compared with what I had gone with (13pt font, no set font-weight):

![withnomockupstyles](https://user-images.githubusercontent.com/6207644/63541167-84730580-c4eb-11e9-906f-379d6fd08e0b.png)


to my eyes that's *visually* closer to the mockups although it doesn't match the numerical specs. What do you think we should go with?

Reply:
here it is with 13pt font and font-weight: 500 (I also realized I forgot to fix the top margin on the title):

![paddingand500](https://user-images.githubusercontent.com/6207644/63541377-f3e8f500-c4eb-11e9-925f-129ba8d37aab.png)

I think that looks a little closer to the mockups, what do you think?

Reply:
last one looks good to me